data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/RateLimiterAggregateActionConfig.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-Limense-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.processor.aggregate.actions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
+
+public class RateLimiterAggregateActionConfig {
+    @JsonProperty("events_per_second")
+    @NotNull
+    int eventsPerSecond;
+    
+    @JsonProperty("drop_when_exceeds")
+    boolean dropWhenExceeds = false;
```

Comment:
I'm fine with keeping this as `false` if we want blocking to be the default behavior. Which do think is the ideal default?

If dropping should be default, then we could have `block_when_exceeds: true` to change the behavior.